src/locales/fr.json

```diff
@@ -404,13 +404,19 @@
   "Notifications": {
     "title": "Notifications",
     "description": "Vous recevrez les notifications par email ou par notification mobile si vous installez l'app Cozy Banks sur votre mobile",
+    "editModal": {
+      "title": "Configurer la notification"
+    },
     "if_balance_lower": {
       "settingTitle": "Solde bas",
+      "fieldLabels": {
+        "value": "Me prévenir quand mon solde passe sous"
+      },
       "email": {
         "innerTitle": "Notification de solde bas",
         "access_accounts": "Accéder à mes comptes"
       },
-      "description": "Vous recevrez une notification, si votre solde est inférieur à",
+      "description": "Vous recevrez une notification, si votre solde est inférieur à *%{value}*€",
```

Comment:
There's no comma in the english version, and I think it's better to read without it.